Update README to include Contributing information

three opened the pull request:
With the change to include Travis the README should reflect how to use the new system. Information about testing and rules for commits have been added to guide contributors.

three committed:
Update README to include Contributing information

winstliu commented on README.md:
```diff
@@ -1,4 +1,60 @@
+
+All members of engineering club are free to contribute to the MHS Engineering
+Club website. When contributing, we want the repository to stay as neat and
+clean as possible so please follow the following rules:
```
Looks like there's a missing comma between `possible` and `please`

winstliu commented on README.md:
```diff
@@ -1,4 +1,60 @@
+Club website. When contributing, we want the repository to stay as neat and
+clean as possible so please follow the following rules:
+
+- **Commit using pull requests.** If you want to commit, create a branch and
```
This point might not be needed since pushing to master is already forbidden.

winstliu commented on README.md:
```diff
@@ -1,4 +1,60 @@
+  yourself. Make sure to check all pages that could be affected by your
+  change. For instance, if you alter style.css, you must check every html
+  page that links to style.css.
+- **Don't rely on /page to link to /page.html.** Github supports it, but most
```
Wrap the code in `s, not *s (eg `/page` instead of _/page_)

winstliu commented on README.md:
```diff
@@ -1,4 +1,60 @@
+Python provides an HTTP module that will run a simple HTTP server from the
+current directory. This is started differently depending on what version of
+python you're using.
+    $ python2 -m SimpleHTTPServer
```
Triple backticks instead of indentation for clarity here maybe?

three committed:
Fix formatting in README

Files are wrapped in back ticks instead of asterisks for clarity. Grammatical
changes made where necessary.

winstliu commented:
Don't forget CONTRIBUTING.md.
